src/app/app.component.ts

```diff
@@ -1,57 +1,56 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
 import * as Sentry from "@sentry/browser";
 
-@Component({
```

Comment:
This was moved below the `block()` function definition but no other changes made